tutorials/platform/android/android_plugin.rst

```diff
@@ -1,165 +1,388 @@
-:article_outdated: True
-
 .. _doc_android_plugin:
 
-Creating Android plugins
-========================
+Godot Android plugins
+=====================
 
 Introduction
 ------------
 
 Android plugins are powerful tools to extend the capabilities of the Godot engine
-by tapping into the functionality provided by the Android platform and ecosystem.
-
-Mobile gaming monetization is one such example since it requires features
-and capabilities that don't belong to the core feature set of a game engine:
-
-- Analytics
-- In-app purchases
-- Receipt validation
-- Install tracking
-- Ads
-- Video ads
-- Cross-promotion
-- In-game soft & hard currencies
-- Promo codes
-- A/B testing
-- Login
-- Cloud saves
-- Leaderboards and scores
-- User support & feedback
-- Posting to Facebook, Twitter, etc.
-- Push notifications
+by tapping into the functionality provided by Android platforms and ecosystem.
+
+For example in Godot 4, Android plugins are used to support multiple Android-based
+XR platforms without encumbering the core codebase with vendor specific code or binaries.
 
 Android plugin
 --------------
 
-While introduced in Godot 3.2, the Android plugin system got a significant architecture update starting with Godot 3.2.2.
-The new plugin system is backward-incompatible with the previous one and in Godot 4.0, the previous system was fully deprecated and removed.
-Since we previously did not version the Android plugin systems, the new one is now labelled ``v1`` and is the starting point for the modern Godot Android ecosystem.
+**Version 1 (v1)** of the Android plugin system was introduced in Godot 3.2.2. That version allowed developers
+to augment the Godot 3 engine with Java / Kotlin logic as well as GDNative functionality.
```

Comment:
Again, does it make sense to reference "Godot 3.2.2" and "GDNative" in the context of the Godot 4.x docs? Perhaps this should just talk about v1 being used in Godot 4.0 and 4.1?

Reply:
This section is to give some background as to the reason for the `v2` versioning.
But to your point, I rephrase it to mention compatibility with Godot 4.0 and 4.1 and remove reference to GDNative.

Reply:
Looks good!